src/lib/components/layout/FormLayout/FormLayoutCustomField.jsx

```diff
@@ -112,10 +115,6 @@ FormLayoutCustomField.propTypes = {
    * Optional ID of labeled field to keep accessibility features. Only available if `label` is set.
    */
   labelForId: PropTypes.string,
-  /**
-   * Layout of the field, controlled by parent FormLayout.
-   */
-  layout: PropTypes.oneOf(['horizontal', 'vertical']),
```

Comment:
We still want to have option to override it locally. For example, I might want to just have one totally separate field with different layout and It is non-sense to use FormLayoutProvider in this case. It applies for all following occurrences.

Reply:
For info. This comment applies only to `layout`, it does not apply for `inFormLayout`.

Reply:
Current implementation does not support it. And I did not want to introduce BC changes in this PR.

I don't have an opinion about this in general, but since it would cause a change of behaviour I would prefer to not do it in this PR.

@adamkudrna Is it even technically possible?

Reply:
I don't get it. You are the one who is creating BC by removing option to locally overwrite `layout` ... or am I wrong?

Reply:
While before this PR it was possible to set `layout` for a component within `FormLayout` it did nothing - it was ignored.